src/ngx_http_lua_shdict.c

```diff
@@ -36,11 +36,20 @@ static int ngx_http_lua_shdict_flush_expired(lua_State *L);
 static int ngx_http_lua_shdict_get_keys(lua_State *L);
 
 
+static ngx_inline ngx_shm_zone_t* ngx_http_lua_shdict_get_zone(lua_State *L,
+                                                           int index);
```

Comment:
Style: the parameter declarations need to align with the first argument in the previous line.